dipy/workflows/stats.py

```diff
@@ -66,16 +66,15 @@ def run(self, data_files, bvals_files, bvecs_files, mask_file,
             Path of a brain mask file.
         bbox_threshold : variable float, optional
             Threshold for bounding box, values separated with commas for ex.
-            [0.6,1,0,0.1,0,0.1]. (default (0.6, 1, 0, 0.1, 0, 0.1))
+            [0.6,1,0,0.1,0,0.1].
         out_dir : string, optional
-            Where the resulting file will be saved. (default '')
+            Where the resulting file will be saved. (default current directory)
         out_file : string, optional
-            Name of the result file to be saved. (default 'product.json')
+            Name of the result file to be saved.
         out_mask_cc : string, optional
-            Name of the CC mask volume to be saved (default 'cc.nii.gz')
+            Name of the CC mask volume to be saved.
```

Comment:
Unlike in here, for may of the modified docstrings a period was not added. Please add it consistently for the modified docstrings where it is missing.

Reply:
Addressed in d2ab2d4